tools/src/bls/main.rs

```diff
@@ -16,8 +16,4 @@ fn main() {
     println!("# Secret Key:");
     println!();
     println!("{}", hex::encode(secret_key.serialize_to_vec()));
-    println!();
-    println!("# Proof Of Knowledge:");
-    println!();
-    println!("{}", hex::encode(&secret_key.sign(&public_key).compress()));
-}
```

Comment:
Why did you remove the proof of knowledge?

Reply:
That's a good question. I don't remember doing that.
I'll add it back.

Reply:
And done!